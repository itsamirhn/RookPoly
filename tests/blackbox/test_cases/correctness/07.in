4
10101
001
00111
100
