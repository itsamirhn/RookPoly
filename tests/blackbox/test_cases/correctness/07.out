2X^4 + 12X^3 + 17X^2 + 8X + 1
